scripts/sysman/common.yml

```diff
@@ -211,6 +211,10 @@ etors:
     - name: DEVICE_UUID
       desc: $s_uuid_t
       version: "1.7"
+    - name: MEMORY_STATE_EXP
```

Comment:
Per the suggestion by @joshuaranjan in the conversation, perhaps a more specific name such as `MEMORY_PAGE_OFFLINE_STATE_EXP` would be preferable, for both the name and the structure type.

Reply:
Would this create a new data structure instead of the extension to the original mem_state structure?

Reply:
@vaidyan1-int no, I think ask here was to just rename your extension proposal to be more specific. (something like `zes_memory_page_offline_state_exp`) rather than simply `zes_memory_state_exp`. We can always expose multiple extensions via the pNext in existing zes_mem_state_t. if we keep struct names as specific now, it would help to differentiate any new extensions we might need to add here

Reply:
Got it. I understand the intent. Will this also initiate change to the yml file name to memPageOfflineState.yml ?

Reply:
yes, would be good to rename file as well. thanks!

Reply:
Done 👍